README.rst

```diff
@@ -9,9 +9,8 @@ aiokatcp
    :target: http://aiokatcp.readthedocs.io/en/latest/
 
 aiokatcp is an implementation of the `katcp`_ protocol based around the Python
-asyncio system module. It requires Python 3.5 or later, as it makes extensive
-uses of coroutines and type annotations. It is loosely inspired by the `Python
-2 bindings`_, but has a much narrower scope.
+asyncio system module. It requires Python 3.6 or later. It is loosely inspired
+by the `Python 2 bindings`_, but has a much narrower scope.
 
 .. _katcp: https://katcp-python.readthedocs.io/en/latest/_downloads/NRF-KAT7-6.0-IFCE-002-Rev5.pdf
```

Comment:
It seems this link doesn't work anymore.  Also, there is a Rev 5.1 doc available.

https://katcp-python.readthedocs.io/en/latest/_downloads/361189acb383a294be20d6c10c257cb4/NRF-KAT7-6.0-IFCE-002-Rev5-1.pdf

Reply:
Thanks for spotting that @ajoubertza. I've sent you a separate PR for it.